packages/contact-center/store/src/store.ts

```diff
@@ -1,14 +1,6 @@
 import {makeAutoObservable, observable} from 'mobx';
 import Webex from 'webex';
-import {
-  AgentLogin,
-  IContactCenter,
-  Profile,
-  Team,
-  WithWebex,
-  InitParams,
-  IStore
-} from './store.types';
+import {AgentLogin, IContactCenter, Profile, Team, WithWebex, InitParams, IStore} from './store.types';
```

Comment:
The below are the prettier fix, on saving the file it fixes the prettier issues.